# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/ipc/GPUProcessHost.cpp
# Commit: 1d9384906287
# Full Hash: 1d93849062870a8d492eeef2150ecea174cb6223
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-26 21:12:53
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r f5ebdcca640d -r 1d9384906287 gfx/ipc/GPUProcessHost.cpp
--- a/gfx/ipc/GPUProcessHost.cpp	Sun May 26 14:29:56 2019 +0000
+++ b/gfx/ipc/GPUProcessHost.cpp	Sun May 26 14:30:14 2019 +0000
@@ -6,10 +6,10 @@
 
 #include "GPUProcessHost.h"
 #include "chrome/common/process_watcher.h"
-#include "gfxPrefs.h"
 #include "mozilla/gfx/Logging.h"
 #include "nsITimer.h"
 #include "mozilla/Preferences.h"
+#include "mozilla/StaticPrefs.h"
 #include "VRGPUChild.h"
 #include "ProcessUtils.h"
 
@@ -62,7 +62,7 @@
     return !!mGPUChild;
   }
 
-  int32_t timeoutMs = gfxPrefs::GPUProcessTimeoutMs();
+  int32_t timeoutMs = StaticPrefs::GPUProcessTimeoutMs();
 
   // If one of the following environment variables are set we can effectively
   // ignore the timeout - as we can guarantee the compositor process will be